Unable to make on new Pi Zero

See [#ballle98/57 ](https://github.com/ballle98/AqualinkD/issues/57)

compiles fine on my older Pi 2B+ but on a newer Pi 4 64-bit it is not building

```
$ make
OS: 9 (stretch)
GLIBC: ldd (Debian GLIBC 2.24-11+deb9u4) 2.24
...
MD5 -D MG_DISABLE_HTTP_DIGEST_AUTH -D MG_DISABLE_MD5 -D MG_DISABLE_JSON_RPC  -c iaqtouch_aq_programmer.c  -o iaqtouch_aq_programmer.o
./release/aqualinkd has been compiled
gcc -Wall -O3  -D AQ_PDA -D AQ_ONETOUCH -D AQ_IAQTOUCH -D AQ_RS16 -D MG_DISABLE_MD5 -D MG_DISABLE_HTTP_DIGEST_AUTH -D MG_DISABLE_MD5 -D MG_DISABLE_JSON_RPC  -o ./release/aqualinkd aqualinkd.o utils.o config.o aq_serial.o aq_panel.o aq_programmer.o net_services.o net_services_habridge.o json_messages.o rs_msg_utils.o devices_jandy.o packetLogger.o devices_pentair.o color_lights.o serialadapter.o mongoose.o timespec_subtract.o pda.o pda_menu.o pda_aq_programmer.o onetouch.o onetouch_aq_programmer.o iaqtouch.o iaqtouch_aq_programmer.o -l pthread -l m
./release/aqualinkd has been compiled

```


Newer Pi 4

```
$ make
OS: 11 (bullseye)
GLIBC: ldd (Debian GLIBC 2.31-13+rpt2+rpi1+deb11u2) 2.31
...
/usr/bin/ld: onetouch.o:(.bss+0x0): multiple definition of `_menu'; pda_menu.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:121: release/aqualinkd] Error 1


```

--- a/onetouch.c
+++ b/onetouch.c
@@ -35,11 +35,11 @@
 //#include "pda_menu.h"
 
 
-int _ot_hlightindex = -1;
-int _ot_hlightcharindexstart = -1;
-int _ot_hlightcharindexstop = -1;
-char _menu[ONETOUCH_LINES][AQ_MSGLEN+1];
-struct ot_macro _macros[3];
+static int _ot_hlightindex = -1;
+static int _ot_hlightcharindexstart = -1;
+static int _ot_hlightcharindexstop = -1;
+static char _menu[ONETOUCH_LINES][AQ_MSGLEN+1];
+static struct ot_macro _macros[3];
 
 void set_macro_status();
 void pump_update(struct aqualinkdata *aq_data, int updated);

--- a/pda_menu.c
+++ b/pda_menu.c
@@ -25,8 +25,8 @@
 #include "aq_serial.h"
 #include "utils.h"
 
-int _hlightindex = -1;
-char _menu[PDA_LINES][AQ_MSGLEN+1];
+static int _hlightindex = -1;
+static char _menu[PDA_LINES][AQ_MSGLEN+1];
 
 void print_menu()
 {

